### DISCUSSION TITLE
Welcome to tesla-local-control-addon Discussions!

### DISCUSSION THREAD

**Discussion by raphmur** [tesla-local-control/tesla-local-control-addon#113] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by iainbullock** [comment]
@raphmur thanks for setting this up. I'll introduce myself. I'm Iain Bullock (@iainbullock) one of the project Maintainers. Here's some information regarding my setup:

- I'm running the https://github.com/tesla-local-control/tesla_ble_mqtt_docker version of the project on a Raspberry Pi 3+. The RPi runs Raspberry Pi OS Lite 64bit (release 2024-07-04)
- I run the Docker version of Home Assistant on a Macmini A1103 running Ubuntu Server 22.04.4 LTS
- I have a 2024 Model 3 RWD (standard range with 60kWh battery)
- I have 1.6kW of solar panels, and a 9.6kWh / 3kW storage battery

My main use case is to adjust the car's Charging Current in response to excess solar generation via a Home Assistant automation. I also have various cards / automations to control various functions on the car using a combination of this project and https://github.com/alandtse/tesla (using https://github.com/iainbullock/tesla-http-proxy-docker as the Fleet API proxy)
